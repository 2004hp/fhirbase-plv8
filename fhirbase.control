# jsquery extension
comment = 'FHIR database'
default_version = '1.0'
module_pathname = '$libdir/fhirbase'
relocatable = true

