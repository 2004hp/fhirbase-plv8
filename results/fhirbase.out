CREATE EXTENSION fhirbase;
ERROR:  syntax error at or near "\"
SET escape_string_warning=off;
SELECT * FROM patient;
ERROR:  relation "patient" does not exist
LINE 1: SELECT * FROM patient;
                      ^
