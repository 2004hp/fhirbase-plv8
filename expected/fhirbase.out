CREATE EXTENSION fhirbase;
SET escape_string_warning=off;
SELECT * FROM patient;
--
(0 rows)

